File: apkEditor/src/main/java/com/mcal/apkeditor/ui/fulleditor/ResourcesFragment.java
package com.mcal.apkeditor.ui.fulleditor;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import com.mcal.apkeditor.databinding.FragmentResourcesBinding;

public class ResourcesFragment extends Fragment {
    private FragmentResourcesBinding binding;

    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        binding = FragmentResourcesBinding.inflate(inflater, container, false);
        View root = binding.getRoot();

        return root;
    }
}

File: apkEditor/src/main/java/com/mcal/apkeditor/ui/fulleditor/resources/ResourcesFragment.kt
package com.mcal.apkeditor.ui.fulleditor.resources

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.fragment.app.activityViewModels
import com.mcal.apkeditor.databinding.FragmentResourcesBinding
import com.mcal.apkeditor.ui.fulleditor.FullEditorViewModel

class ResourcesFragment : Fragment() {
    private lateinit var binding: FragmentResourcesBinding
    private val model: FullEditorViewModel by activityViewModels()

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        binding = FragmentResourcesBinding.inflate(inflater, container, false)
        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
    }
}